keps/sig-network/2079-network-policy-port-range/README.md

```diff
@@ -266,7 +265,7 @@ _This section must be completed when targeting beta graduation to a release._
   The increase of 5xx http error count on Network Policies Endpoint
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  This will be done.
+  Yes, with unit tests
```

Comment:
Can you please run manual tests on a real cluster before switching on FG by default.

[You can do that after feature freeze and update the KEP then. You can see an example of that here:
https://github.com/kubernetes/enhancements/pull/2538]

So my suggestion for now is to change to:
Only with unit tests. End-to-end tests will done before enabling feature-gate by default.